whois-commons/src/main/resources/internals_data.sql

```diff
@@ -22,4 +22,6 @@ INSERT INTO authoritative_resource (source, resource) VALUES ('test', '0.0.0.0/0
 -- legacy autnums
 INSERT INTO legacy_autnums (autnum) VALUES ('103');
 INSERT INTO legacy_autnums (autnum) VALUES ('12666');
-INSERT INTO legacy_autnums (autnum) VALUES ('12668');
\ No newline at end of file
+INSERT INTO legacy_autnums (autnum) VALUES ('12668');
+
+INSERT INTO transfer_update_lock VALUES (0);
```

Comment:
Maybe add a comment on top of this? To follow the same format that rest of INSERTS?